client/signup/steps/import-url/index.jsx

```diff
@@ -97,9 +96,7 @@ class ImportURLStepComponent extends Component {
 				} ) => {
 					if ( ! includes( importerTypes, 'url' ) ) {
 						return this.setUrlError(
-							translate(
-								"That doesn't seem to be a Wix or GoDaddy site. Please check the URL and try again."
-							)
+							translate( "That doesn't seem to be a Wix site. Please check the URL and try again." )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.